.github/workflows/ci.yml

```diff
@@ -8,186 +8,44 @@ on:
 
 jobs:
   tests:
-    name: PHP ${{ matrix.php }}, SF ${{ matrix.symfony }} - ${{ matrix.deps }}
+    name: PHP ${{ matrix.php }}, SF ${{ matrix.symfony }} - ${{ matrix.deps }} - ${{ matrix.use-orm == '1' && 'ORM, ' || '' }} ${{ matrix.use-odm == '1' && 'ODM, ' || '' }} ${{ matrix.use-dama == '1' && 'DAMA' || '' }}
```

Comment:
This is nice!

Reply:
`CI / PHP 8, SF 5.4.* - highest - ORM, ` => can be improved :smile: